### ISSUE TITLE
Do not check delta when swiping in progress

### ISSUE THREAD

**Issue by mctep** [dogfessional/react-swipeable#74]
Hello!! Thank you for awesome lib! It is great.

But I have suggestion to improve some case. I want to make `swipe to delete` like in gmail mobile app.

```jsx
<Swipeable
  onSwiped={this.handleSwiped}
  onSwipingLeft={this.handleSwipingLeft}
  onSwipingRight={this.handleSwipingRight}
/>
```

![before](https://cloud.githubusercontent.com/assets/1949681/26715748/842b53de-477f-11e7-9c51-ed69d60ba50c.gif)

As you can see after I move finger back to the start point there is some lag when I change swiping direction.

In my case it can be fixed with changes in this [line](https://github.com/dogfessional/react-swipeable/blob/1f46f6f76c9749e224de37944bf99a9a92dc4965/src/Swipeable.js#L120):

```js
if (pos.absX < delta && pos.absY < delta && !this.swipeable.swiping) return;
```

**Comment by hartzis** [comment]
hey @mctep,

seems reasonable, would you like to make a PR with a simple test case added for it? If not i should be able to get around to it within the next couple of weeks.

Should be able to be a minor version bump too i think.

Cheers!

**Comment by hartzis** [comment]
closed by [v4.0.1](https://github.com/dogfessional/react-swipeable/releases/tag/v4.0.1)
